integration-tests/smallrye-jwt-with-jwt-build/pom.xml

```
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-integration-tests-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-integration-test-smallrye-jwt-with-jwt-build</artifactId>
+    <name>Quarkus - Integration Tests - Smallrye JWT and JWT Build</name>
+    <description>Module that contains Smallrye JWT and JWT Build tests</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy</artifactId>
```

```suggestion
            <artifactId>quarkus-rest</artifactId>
```
It's preferred stack. Unless you have a good reason to go with RESTEasy Classic.